File: mse_team2/Assets/Scripts/Gamehistory related/GameResultInfo.cs
using System;
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

// class for game result information
// this will be used when the game result is sent to server part after game ends
[Serializable]
public class GameResultInfo : MonoBehaviour
{
    public long privateCode;
    public string gameMode;
    public string result;

    public void setPrivateCode(long privateCode){
        this.privateCode = privateCode;
    }
    public void setGameMode(string gameMode){
        this.gameMode = gameMode;
    }
    public void setResult(string result){
        this.result = result;
    }
}

File: FarmWars/src/main/java/com/example/demo/GameResultInfo.java
package com.example.demo;

public class GameResultInfo {
	private long privateCode;
	private String gameMode;
	private String result;
	
	public GameResultInfo(long privateCode, String gameMode, String result) {
		super();
		this.privateCode = privateCode;
		this.gameMode = gameMode;
		this.result = result;
	}
	
	public long getPrivateCode() {
		return privateCode;
	}
	
	public String getGameMode() {
		return gameMode;
	}
	
	public String getResult() {
		return result;
	}
}
